### PR TITLE
Support flank version 8.1.0

### PR DESCRIPTION
**Pull request by tahirhajizada**
Add new feature from flank version 8.1.0.
`keep-file-path`
Keeps the full path of downloaded files from a Google Cloud Storage bucket. Required when file names are not unique. Disabled by default.

### CODE REVIEW

**Comment by runningcode** [comment]
Hey, thanks for the contribution. Can we add some documentation to the readme around this flag and what it does?

**Comment by tahirhajizada** [comment]
@runningcode Yes sure. I was going to add it.

**Comment by tahirhajizada** [comment]
@runningcode I see there is already one open PR with version `0.7.2-SNAPSHOT`, in this case which version should I put?

File: README.md [hunk 1]
```diff
@@ -173,6 +174,9 @@ Monitor and record performance metrics: CPU, memory, network usage, and FPS (gam
 ### resultsBucket
 The name of a Google Cloud Storage bucket where raw test results will be stored.
 
+### keepFilePath
+Keeps the full path of downloaded files from a Google Cloud Storage bucket. Required when file names are not unique. Disabled by default.
```
Comment (runningcode) [inline README.md]: what file names would not be unique? how does one know that one should enable this?

Comment (tahirhajizada) [reply README.md]: As an example. When the toggle is disabled downloaded files are stored at the root of device.
`results/2019-09-20-13-57-45.1454_nGys/shard_0/NexusLowRes-28-en-portrait/file.png`

But when it is enabled, then the full path of the file kept.
`results/2019-09-20-13-57-45.1454_nGys/shard_0/NexusLowRes-28-en-portrait/{package name}/file.png`

**Pull request merged**